bindings/python/src/metadata.rs

```diff
@@ -90,6 +90,17 @@ impl Metadata {
     pub fn mode(&self) -> EntryMode {
         EntryMode(self.0.mode())
     }
+    /// Last modified time
+    #[getter]
+    pub fn last_modified(&self, py: Python) -> PyResult<Option<PyObject>> {
```

Comment:
Please return a `chrono::DateTime<Tz>` directly here; `pyo3` will convert it to Python's `datetime.datetime`. While `PyObject` also works, its API semantics are unclear.

https://pyo3.rs/v0.24.0/conversions/tables.html

Reply:
Yeah, I know about this, but it makes reading the code a bit harder. I prefer to specify it more explicitly.

Reply:
after some search PYO3 did most of the dirty things for convert include the Tz things in chrono

use  chrono here maybe bring a little complexity(most for the Tz for type) 

maybe PyO3 can make the chore::datetime to pub?
cc messense
more:

https://github.com/PyO3/pyo3/pull/4791/files

Reply:
> use chrono here maybe bring a little complexity(most for the Tz for type)

We need to us Tz, always use UTC instead.

Reply:
> > use chrono here maybe bring a little complexity(most for the Tz for type)
> 
> We don't need to us Tz, always use UTC instead.

cool! will do this. Also think PyO3 can pub this ~

Reply:
Done